src/Dao/PlagiabotDao.php

```diff
@@ -48,7 +48,7 @@ public function __construct( $dsn, $user, $pass, $wiki = 'https://en.wikipedia.o
 
 	/**
 	 * @param int $n Number of records asked for
-	 * @param string $filter Filter SQL to show a certian status, one of 'all', 'fixed', 'noaction', 'open' or 'mine'
+	 * @param string $filter Filter SQL to show a certian status, one of 'all', 'open', 'reviewed' or 'mine'
 	 * @param string $filterUser Filter SQL to only return records reviewed by given user
```

Comment:
This PHPDoc block should be updated now that we're passing options array instead of filter and filterUser (sorry, I should have done this in my fix.)